css/animal_selector.styl

```diff
@@ -288,13 +288,14 @@
     background: rgba($black, 0.33);
     border-radius: 3px;
     box-shadow: 0 2px 3px rgba($black, 0.5) inset;
-    overflow: auto;
+    overflow: hidden;
```

Comment:
Not sure why this is showing as a diff. As you can see from https://github.com/zooniverse/Serengeti/blob/master/css/animal_selector.styl#L291 master already has the 'hidden' value.